tests/0.8.1/invalid.json

```diff
@@ -0,0 +1,842 @@
+    "assertions": {
+      "header": {
+        "alg": "EdDSA",
+        "typ": "JWT",
+        "ucv": "0.8.1"
+      },
+      "validationErrors": [
+        "payloadMissingOrInvalid"
+      ]
+    }
+  },
+  {
+    "comment": "UCAN signature is missing or invalid",
+    "token": "eyJhbGciOiJFZERTQSIsInR5cCI6IkpXVCIsInVjdiI6IjAuOC4xIn0.eyJpc3MiOiJkaWQ6a2V5Ono2TWtnYUFhVlN6RTlYcWFucmJGRDFKOVJHVEZjV1U3TjlCUGdqU1dtMTdwazFtQiIsImF1ZCI6ImRpZDprZXk6ejZNa3JqekF5TlFWMkhtRlJpTGhOb2lUVzdSWjc2SlVUbnVIdGZzVE4xUks2WlBUIiwiZXhwIjo0ODAzODkwNTUyLCJhdHQiOltdLCJwcmYiOltdfQ",
+    "assertions": {
+      "header": {
+        "alg": "EdDSA",
+        "typ": "JWT",
+        "ucv": "0.8.1"
+      },
+      "payload": {
+        "iss": "did:key:z6MkgaAaVSzE9XqanrbFD1J9RGTFcWU7N9BPgjSWm17pk1mB",
+        "aud": "did:key:z6MkrjzAyNQV2HmFRiLhNoiTW7RZ76JUTnuHtfsTN1RK6ZPT",
+        "exp": 4803890552,
+        "att": [],
+        "prf": []
+      },
+      "validationErrors": [
+        "signatureMissingOrInvalid"
+      ]
+    }
+  },
+  {
+    "comment": "UCAN has expired",
+    "token": "eyJhbGciOiJFZERTQSIsInR5cCI6IkpXVCIsInVjdiI6IjAuOC4xIn0.eyJpc3MiOiJkaWQ6a2V5Ono2TWtnYUFhVlN6RTlYcWFucmJGRDFKOVJHVEZjV1U3TjlCUGdqU1dtMTdwazFtQiIsImF1ZCI6ImRpZDprZXk6ejZNa3JqekF5TlFWMkhtRlJpTGhOb2lUVzdSWjc2SlVUbnVIdGZzVE4xUks2WlBUIiwiZXhwIjoxNjQ3Nzg0OTUyLCJhdHQiOltdLCJwcmYiOltdfQ.EvUWrIbS0_3QKiL5JJT2N6TejC8h6u-weydGRozCfcsbCEiAGEYa1RC1Yx06pekdrxqveAkQMNbvcb4s6PhAAA",
+    "assertions": {
+      "header": {
+        "alg": "EdDSA",
+        "typ": "JWT",
+        "ucv": "0.8.1"
+      },
+      "payload": {
+        "iss": "did:key:z6MkgaAaVSzE9XqanrbFD1J9RGTFcWU7N9BPgjSWm17pk1mB",
+        "aud": "did:key:z6MkrjzAyNQV2HmFRiLhNoiTW7RZ76JUTnuHtfsTN1RK6ZPT",
+        "exp": 1647784952,
+        "att": [],
+        "prf": []
+      },
+      "validationErrors": [
+        "expExpired"
+      ]
+    }
+  },
+  {
+    "comment": "UCAN is not ready to be used",
+    "token": "eyJhbGciOiJFZERTQSIsInR5cCI6IkpXVCIsInVjdiI6IjAuOC4xIn0.eyJpc3MiOiJkaWQ6a2V5Ono2TWtnYUFhVlN6RTlYcWFucmJGRDFKOVJHVEZjV1U3TjlCUGdqU1dtMTdwazFtQiIsImF1ZCI6ImRpZDprZXk6ejZNa3JqekF5TlFWMkhtRlJpTGhOb2lUVzdSWjc2SlVUbnVIdGZzVE4xUks2WlBUIiwibmJmIjo0ODAzODkwNTUyLCJleHAiOjQ4MzU0MjY1NTIsImF0dCI6W10sInByZiI6W119.Yoa2Dslj8HaVjhzDNfhpQ5Qp2V6mydb0I48RcfLrEmjUQj9rO2sKYOUmkQUg6pCODLfo6SjT3aEW6GiK2yccDQ",
+    "assertions": {
+      "header": {
+        "alg": "EdDSA",
+        "typ": "JWT",
+        "ucv": "0.8.1"
+      },
+      "payload": {
+        "iss": "did:key:z6MkgaAaVSzE9XqanrbFD1J9RGTFcWU7N9BPgjSWm17pk1mB",
+        "aud": "did:key:z6MkrjzAyNQV2HmFRiLhNoiTW7RZ76JUTnuHtfsTN1RK6ZPT",
+        "nbf": 4803890552,
+        "exp": 4835426552,
+        "att": [],
+        "prf": []
+      },
+      "validationErrors": [
+        "nbfNotReady"
+      ]
+    }
+  },
+  {
+    "comment": "Witnesses expire before the delegated UCAN",
```

Comment:
The expiration times for the delegate and witness are `5435042552`. Did you mean edit the witness forward in time for this test case?